tests/benchmarks/test_snowflake.py

```diff
@@ -0,0 +1,58 @@
+import os
+import uuid
+
+import dask
+import dask.dataframe as dd
+import pytest
+from dask_snowflake import read_snowflake, to_snowflake
+from distributed import Client
+from snowflake.sqlalchemy import URL
+from sqlalchemy import create_engine
+
+
+@pytest.fixture
```

Comment:
Not sure I follow why are we creating a local cluster and client here.

Reply:
Oops -- we don't need that.